src/client/clipboard.cpp

```diff
@@ -0,0 +1,90 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <string>
+#include <memory>
+#include "clipboard.h"
+
+#define XCLIP_SELECT "/usr/bin/xclip -selection c"
```

Comment:
Maybe i should use a bash script or sort to check before hand and figure out where the location is? Or is that a bad approach.